src/tendencies/implicit/implicit_tendency.jl

```diff
@@ -122,6 +122,8 @@ function implicit_vertical_advection_tendency!(Yₜ, Y, p, t, colidx)
                 dt,
                 edmfx_upwinding,
             )
+            @. Yₜ.c.sgsʲs.:($$j).ρae_tot[colidx] -=
```

Comment:
Lets think on how to ensure that the `Y_t` has all the tendencies that we need it to have. Should we cache it? Maybe at least we could leave a comment about what order of updates we are assuming.